apps/meteor/server/models/raw/Rooms.js

```diff
@@ -75,13 +75,16 @@ export class RoomsRaw extends BaseRaw {
 					},
 			  };
 
+		const fnameQuery = useFname ? [{ fname: nameRegex }] : [];
```

Comment:
shouldn't it be like this?
```suggestion
		const fnameQuery = useFname ? { fname: nameRegex } : {};
```